lib/carin_for_blue_button_test_kit/docs/carin_for_blue_button_v2.0.0_client_suite_description.md

```diff
@@ -0,0 +1,76 @@
+on the tests. Future versions of these tests may verify other
+requirements and may change the test verification logic.
+
+## Test Methodology
+
+For these tests Inferno simulates a FHIR server playing the role of a 
+[CARIN IG for Blue Button® Responder](https://hl7.org/fhir/us/carin-bb/STU2/Conformance_Requirements.html#actors). Inferno will wait for
+the client under test to request a resource for each of the CARIN for Blue Button profiles, which include the following:
+  - [Patient](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-Patient.html)
+  - [Coverage](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-Coverage.html)
+  - [Organization](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-Organization.html)
+  - [Practitioner](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-Practitioner.html)
+  - [RelatedPerson](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-RelatedPerson.html)
+  - [Inpatient Institutional ExplanationOfBenefit](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-ExplanationOfBenefit-Inpatient-Institutional.html)
+  - [Oral ExplanationOfBenefit](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-ExplanationOfBenefit-Oral.html)
+  - [Outpatient Institutional ExplanationOfBenefit](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-ExplanationOfBenefit-Outpatient-Institutional.html)
+  - [Pharmacy ExplanationOfBenefit](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-ExplanationOfBenefit-Pharmacy.html)
+  - [Professional NonClinician ExplanationOfBenefit](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-ExplanationOfBenefit-Professional-NonClinician.html)
+
+The tests also waits for search requests to be made for each of the required searches described for each resource in the
+CARIN for Blue Button IG and the US Core 3.1.1 IG. Inferno's simulated server contains instances of each profile defined
+in the IG which include all of the required and Must Support elements.
+
+Once the tester has finished submitting requests to Inferno from the client under test, Inferno will review the requests performed verify conformance. Specifically, Inferno checks that the client:
+  - Retrieved instances of each [CARIN IG for Blue Button profile](https://hl7.org/fhir/us/carin-bb/STU2/artifacts.html#structures-resource-profiles)
+  - Performed searches using the search parameters and search parameter combinations marked as **SHALL** within the [C4BB CapabilityStatement](https://hl7.org/fhir/us/carin-bb/STU2/CapabilityStatement-c4bb.html)
+  and the [US Core Client CapabilityStatement](https://hl7.org/fhir/us/core/STU3.1.1/CapabilityStatement-us-core-client.html).
+  - Processed and retained all profile instances and data elements marked as required or must support via tester attestation.
+
+## Running the Tests
+
+### Quick Start
```

Comment:
Add details about the patient that the client needs to request data for. Assume that they might need to do some kind of search, so I'd include resource id, name, member identifier, date of birth, gender, maybe address.

Reply:
Looks good. Three slight tweaks that may suggest similar tweaks to the data on the reference server:
- formatting: capitalize the first character of each line. I think also bold the title of the data, will just set it off a little nicer.
- include the last name: "example1" is not a great last name. Come up with something realistic, like "Smith".
- for the identifier, I'd put the system in parentheses "(system: https://www.xxxhealthplan.com/fhir/uniquememberidentifier)". I'm not all that happy with making up urls like this. For DTR, we've been prefixing with "http://inferno.healthit.gov/reference-server/r4", so how about changing to http://inferno.healthit.gov/reference-server/r4/uniquememberidentifier in the ref data and using that here? I'm going to try and establish a general policy, but this should be more consistent for now.